README.md

```diff
@@ -74,6 +74,12 @@ $ source environment
 
 #### 4. Build Rust applications
 
+Before building built-in rust examples, you need to build OP-TEE libraries using:
```

Comment:
-> Before building Rust examples

I don't think they are "built-in" examples. We can call the Rust examples built into the OP-TEE filesystem as "built-in" examples, but in this case they are built separately and copied into the filesystem.